misc/test/examples_test.go

```diff
@@ -227,8 +227,25 @@ func TestAccAwsGoAppSync(t *testing.T) {
 	test := getAWSBase(t).
 		With(integration.ProgramTestOptions{
 			Dir: path.Join(getCwd(t), "..", "..", "aws-go-appsync"),
-		})
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				maxWait := 15 * time.Minute
```

Comment:
15 minutes seems like a long time. How long does it typically take for this test to pass? I'd guess 1-2 minutes. Taking that average value and doubling or tripling it should be fine. We can always extend the timeout later. 

Long timeouts should generally be avoided if possible as they extend the overall time it takes for a test run to fail if there actually is an error in the test. (Consider if there is a regression and the ready function continuously fails).

Reply:
Just shortened it to 8 min